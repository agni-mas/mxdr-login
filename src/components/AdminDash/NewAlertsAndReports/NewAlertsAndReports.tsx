import react, { FC } from 'react';
import GridItem from '../../layout/GridItem/GridItem';
import Button from '@mui/material/Button';
import AddIcon from '@mui/icons-material/Add';
import Table from '../../global/table/Table';
import FormGroup from '@mui/material/FormGroup';
import FormControlLabel from '@mui/material/FormControlLabel';
import Checkbox from '@mui/material/Checkbox';
import AlertsAndReports from '../../AnalystDash/AlertsAndReports/AlertsAndReports';


const styles = {
}

interface NewAlertsAndReportsProps {}

const NewAlertsAndReports: FC<NewAlertsAndReportsProps> = () => (
    <GridItem title="New Alerts and Reports">
        <AlertsAndReports />
    </GridItem>
);

export default NewAlertsAndReports;