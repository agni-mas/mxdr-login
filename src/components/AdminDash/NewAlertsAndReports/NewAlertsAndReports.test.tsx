import React from 'react';
import { render } from "@testing-library/react";
import NewAlertsAndReports from './NewAlertsAndReports';


it("renders <NewAlertsAndReports />", () => {
    const { asFragment } = render(<NewAlertsAndReports /> );
    expect(asFragment()).toMatchSnapshot();
});
