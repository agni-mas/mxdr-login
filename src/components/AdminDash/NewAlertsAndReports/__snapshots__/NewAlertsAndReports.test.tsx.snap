// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <NewAlertsAndReports /> 1`] = `
<DocumentFragment>
  <section
    class="item"
  >
    <h2
      class="grid-item-title"
    >
      New Alerts and Reports
    </h2>
    <div
      class="alerts"
    >
      <div
        class="MuiBox-root css-1gsv261"
      >
        <div
          class="MuiTabs-root css-1ujnqem-MuiTabs-root"
        >
          <div
            class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
            style="overflow: hidden; margin-bottom: 0px;"
          >
            <div
              aria-label="basic tabs example"
              class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
              role="tablist"
            >
              <button
                aria-controls="simple-tabpanel-0"
                aria-selected="true"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                color="secondary"
                id="simple-tab-0"
                role="tab"
                tabindex="0"
                type="button"
              >
                Alerts(0)
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                aria-controls="simple-tabpanel-1"
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                id="simple-tab-1"
                role="tab"
                tabindex="-1"
                type="button"
              >
                Reports (0)
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
            <span
              class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
              style="left: 0px; width: 0px;"
            />
          </div>
        </div>
      </div>
      <span
        class="MuiSkeleton-root MuiSkeleton-rectangular MuiSkeleton-wave css-1kcnzpi-MuiSkeleton-root"
        style="height: 400px;"
      />
      <div
        aria-labelledby="simple-tab-1"
        hidden=""
        id="simple-tabpanel-1"
        role="tabpanel"
      />
    </div>
  </section>
</DocumentFragment>
`;
